switch_and_loops/logic_operators.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import "fmt"
+
+func main() {
+        var user_input int
+        fmt.Print("Enter a number:")
+        fmt.Scan(&user_input)
+        switch {
+        case user_input>0:
+                fmt.Print("The number is positive \n")
+                fmt.Printf("Counting down: %d ,", user_input)
+                for user_input!=0 {
+                        user_input-=1
+                        fmt.Print(user_input, ", ")
+                }
+        case user_input<0:
+                for user_input<0 {
+                        fmt.Print("Please enter a positive number: \n")
+                        fmt.Scan(&user_input)
+                        if user_input>=0 {
+                                fmt.Print("The number is positive")
+                                break;
+                        }
+                }
+
+        case user_input==0:
+                fmt.Print("The number is zero")
```

Comment:
```suggestion
                fmt.Println("The number is zero")
```

`Println` will add a `\n` at the end of the string you want to print.